translations.json

```diff
@@ -1505,31 +1509,31 @@
             "en": "Update check for {0} through ACT failed, retrying with GitHub API.",
             "fr-FR": "La vérification de {0} via ACT a échoué. Nouvel essai avec l'API GitHub",
             "ja-JP": "ACTでの {0} のアップデートチェックが失敗しました. GitHub API をリトライします.",
-            "zh-CN": "通过ACT检查{0}的更新失败，正在使用GitHub API重试。"
+            "zh-CN": "通过 ACT 检查 {0} 的更新失败, 正在调用 GitHub API 重试。"
         },
         "ActUpdateException": {
             "#comment": "The first placeholder here is the project being updated (i.e. OverlayPlugin or Cactbot)",
             "de-DE": "ACT Update-Überprüfung für {0} ist fehlgeschlagen, greife auf GitHub API zurück: {1}",
             "en": "ACT Update check for {0} failed, falling back to GitHub API: {1}",
             "fr-FR": "La vérification de la mise à jour ACT {0} a échoué, retour sur l'API GitHub : {1}",
             "ja-JP": "ACTでの {0} のアップデートチェックが失敗しました. GitHub API に戻ります: {1}",
-            "zh-CN": "{0}的ACT检查更新失败，正在回退至GitHub API: {1}"
+            "zh-CN": "{0} 的 ACT 检查更新失败, 正在回退至 GitHub API: {1}"
         },
         "CreatingTempDirFailed": {
             "de-DE": "Fehler beim Erstellen oder Leeren des temporären Verzeichnisses \"{0}\": {1}",
             "en": "Failed to create or empty the temporary directory \"{0}\": {1}",
             "fr-FR": "Impossible de créer ou supprimer le répertoire temporaire \"{0}\": {1}",
             "ja-JP": "一時フォルダの作成に失敗しました \"{0}\": {1}",
             "ko-KR": "임시 디렉토리 \"{0}\"(을/를) 생성하거나 비우지 못했습니다: {1}",
-            "zh-CN": "创建或清空缓存目录\"{0}\": {1}失败"
+            "zh-CN": "创建或清空缓存目录 \"{0}\": {1} 失败"
         },
         "DownloadFailedContentLengthMissing": {
             "de-DE": "Download fehlgeschlagen! Der Download hat keine Dateigröße (Content-Length-Header fehlt).",
             "en": "Download failed! The download has no file size (Content-Length header is missing).",
             "fr-FR": "Le téléchargement a échoué ! Le téléchargement est de taille nulle (l'en-tête Content-Length est manquant).",
             "ja-JP": "ダウンロードに失敗しました! ファイルサイズが不明です (Content-Length が見つかりませんでした).",
             "ko-KR": "다운로드 실패! 다운로드 파일 크기가 없음 (Content-Length 헤더 누락)",
-            "zh-CN": "下载失败！没有下载文件大小（Content-Length标头缺失）。"
+            "zh-CN": "下载失败!丢失下载文件大小 (消息长度标头缺失)。"
```

Comment:
`Content-Length` is a HTTP header field, it would be better to identify the error when not to translate it.